_posts/en/newsletters/2020-05-06-newsletter.md

```diff
@@ -152,8 +152,23 @@ version 0.20.*
     option will give the new features six months to mature before
     their expected release.
 
-- [Bitcoin Core #18038][] P2P: Mempool tracks locally submitted
-  transactions to improve wallet privacy FIXME:bitschmidty
+- [Bitcoin Core #18038][] improves privacy when initially broadcasting a
+  transaction by reducing the frequency that a wallet will attempt to resend
+  from about 30 minutes to about once a day. Previously, an entity surveilling
```

Comment:
suggest s/about once a day/once every 12-36 hours/

Reply:
I prefer 'about once a day'. The exact distribution seems too low-level for a summary write-up, and people who are interested can click through to the PR for full details.